ipam.go

```diff
@@ -182,3 +182,30 @@ func Free(network net.IPNet, mask net.IPMask, subnets []net.IPNet) (net.IPNet, e
 
 	return freeNetwork, nil
 }
+
+// Half takes a networks and returns two subnets which split this network in
+// a half.
+func Half(network net.IPNet) (first, second net.IPNet, err error) {
+	// One bit bigger mask will give a second half of a network.
```

Comment:
This comment doesn't sound right to me, but I couldn't give a better one :D

Reply:
https://github.com/giantswarm/ipam/pull/4/files#diff-a6cb816e9597f944bea1a9807d8633efR194

Reply:
Grand